app/views/user_mailer/invitation_to_claim.html.erb

```diff
@@ -1,19 +1,19 @@
 <% content_for :message do %>
   <h2 class="heading">
-    Hello from the Archive of Our Own!
+    <%= t 'user_mailer.invitation_to_claim.greeting', app_name: ArchiveConfig.APP_NAME %>
   </h2>
 
-  <p>You're receiving this e-mail because an archive has recently been imported by <%= style_link('Open Doors', 'http://opendoors.transformativeworks.org/') %> into the <%= style_link('Archive of Our Own', root_url) %> (AO3), and we believe that the following fanworks belong to you. We'd like to give you a chance to claim (or delete/orphan) these works if you want to. And if you don't already have an account under a different e-mail, we'd like to invite you aboard!</p>
+  <p><%= t 'user_mailer.invitation_to_claim.html.part1', open_doors_link: style_link(t('user_mailer.invitation_to_claim.html.open_doors'), "http://opendoors.transformativeworks.org"), app_short_name: ArchiveConfig.APP_SHORT_NAME, app_link: style_link(ArchiveConfig.APP_NAME, root_url) %></p>
```

Comment:
Just a consistency note: in the previous set of files, the paragraphs were called `thing.para1` but here, they're called `thing.part1`. Might be an idea to pick one style and apply it throughout before this goes further.